src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java

```diff
@@ -208,8 +220,8 @@ public void verifyGalleries(int expected) {
       wait.forElementVisible(galleryNode);
     }
     Assertion.assertNumber(
-            getNumOfElementOnPage(By.cssSelector(".media-gallery-wrapper.ve-ce-branchNode")), expected,
-            "Checking the correct number of gallery nodes");
+        getNumOfElementOnPage(By.cssSelector(".media-gallery-wrapper.ve-ce-branchNode")), expected,
```

Comment:
You can simply move the 
"By.cssSelector(".media-gallery-wrapper.ve-ce-branchNode""

To other By fields in the UI mapping